package.json

```diff
@@ -31,7 +31,7 @@
 		"@popperjs/core": "^2.5.4",
 		"@storybook/addon-actions": "^6.3.12",
 		"@viaa/avo2-components": "^1.76.0",
-		"@viaa/avo2-types": "^2.33.0",
+		"@viaa/avo2-types": "git+ssh://git@github.com:viaacode/avo2-typings.git#b0d46c7",
```

Comment:
Build faalt waarschijnlijk hierop, wat is het process om updates aan de typings te releasen @benjaminnaesen/@bartnaessens?

Reply:
build faalt omdat de oude jenkins file gebruikt wordt , fetch de openshift/Jenkinsfile from master en replace deze in deze branch, 
error : obtained openshift/Jenkinsfile from 27595c71bbd61ac5cdaf7add0be59784140a69f1
Running in Durability level: MAX_SURVIVABILITY
ERROR: Could not find any definition of libraries [jenkins-shared-libs]
org.codehaus.groovy.control.MultipleCompilationErrorsException: startup failed:
WorkflowScript: Loading libraries failed

als je dit dus tegen komt de Jenkinsfile updaten, in de nieuwe jenkins gebruiken we geen shared libs meer